libs/klv/src/test/java/org/codice/alliance/libs/klv/KlvUtilities.java

```diff
@@ -126,6 +128,27 @@ public static KlvIntegerEncodedFloatingPoint createTestFloat(String name, double
     return (KlvIntegerEncodedFloatingPoint) decodedKlvContext.getDataElementByName(name);
   }
 
+  /**
+   * Generates a KlvBytes object from a name and byte array
+   *
+   * @param name name of the klv data element
+   * @param bytes value of the klv data element
+   * @return klv data element
+   * @throws KlvDecodingException
+   */
+  public static KlvBytes createTestBytes(String name, byte[] bytes) throws KlvDecodingException {
+    final byte[] byteArray = new byte[bytes.length + 2];
+    byteArray[0] = -8;
+    byteArray[1] = (byte) bytes.length;
```

Comment:
~Does KLV guarantee that the length of a byte array is less than, uh 128 or maybe 256?~ 
I think line 147 answers my question.